widgetastic_manageiq/expression_editor.py

```diff
@@ -251,8 +263,11 @@ def delete_whole_expression(self):
             self.select_first_expression()
             self.click_remove()
 
-    def read(self):
+    def read(self, complete_expression=False):
         """Returns whole expression as represented visually."""
```

Comment:
**Question**
If we are keeping `complete_expression=False` then this comment should read ""Returns first expression as represented visually"" ?